README.md

```diff
@@ -53,56 +56,26 @@ export GITHUB_REPO=prometheus
     -v GRAFANA_ADMIN_PASSWORD:$GRAFANA_ADMIN_PASSWORD \
     -v GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL:$GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL \
     -v OAUTH_TOKEN="$(printf $OAUTH_TOKEN | base64 -w 0)" \
+    -v GKE_AUTH="$(cat $AUTH_FILE | base64 -w 0)" \
     -v GITHUB_ORG:$GITHUB_ORG -v GITHUB_REPO:$GITHUB_REPO \
     -f manifests/cluster-infra
 ```
+> Note: Use `-v GKE_AUTH="$(echo $AUTH_FILE | base64 -w 0)"` if you're passing the data directly into `$AUTH_FILE`
 - The output will show the ingress IP which will be used to point the domain name to. Alternatively you can see it from the GKE/Services tab.
 - Set the `A record` for `<DOMAIN_NAME>` to point to `nginx-ingress-controller` IP address.
 - The services will be accessible at:
   * Grafana :: `http://<DOMAIN_NAME>/grafana`
   * Prometheus ::  `http://<DOMAIN_NAME>/prometheus-meta`
 
-### Deploy Prow
-> Github integration - monitoring GitHub comments to starts new tests.
-
----
-```
-export HMAC_TOKEN=$(openssl rand -hex 20)
-```
-
-- Add a [github webhook](https://github.com/prometheus/prometheus/settings/hooks) where to send the events.
-  * Content Type: `json`
-  * Send:  `Issue comments,Pull requests`
-  * Secret: `echo $HMAC_TOKEN`
-  * Payload URL: `http://<DOMAIN_NAME>/hook`
-
-- Add all required tokens as k8s secrets.
-  * hmac is used when verifying requests from GitHub.
-  * gke auth is used when scaling up and down the cluster.
-```
-./prombench gke resource apply -a $AUTH_FILE -v ZONE:$ZONE \
-    -v CLUSTER_NAME:$CLUSTER_NAME -v PROJECT_ID:$PROJECT_ID \
-    -v HMAC_TOKEN="$(printf $HMAC_TOKEN | base64 -w 0)" \
-    -v GKE_AUTH="$(cat $AUTH_FILE | base64 -w 0)" \
-    -f manifests/prow/secrets.yaml
-```
-> Note: Use `-v GKE_AUTH="$(echo $AUTH_FILE | base64 -w 0)"` if you're passing the data directly into `$AUTH_FILE`
-
-- Deploy all internal prow components
+### Deploy GitHub Actions
+This section assumes you already have the `main.workflow` file placed to the `.github` directory in the repository from where you want to use prombench from. You can find an example of `main.workflow` in the [`/examples`](examples) directory.
 
+Copy and paste the output of the following to the `AUTH_FILE` secret in the Github Actions workflow.
 ```
-export GITHUB_ORG=prometheus
-export GITHUB_REPO=prometheus
-export PROMBENCH_REPO=https://github.com/prometheus/prombench
-
-./prombench gke resource apply -a $AUTH_FILE -v PROJECT_ID:$PROJECT_ID \
-    -v ZONE:$ZONE -v CLUSTER_NAME:$CLUSTER_NAME -v DOMAIN_NAME:$DOMAIN_NAME \
-    -v GITHUB_ORG:$GITHUB_ORG -v GITHUB_REPO:$GITHUB_REPO \
-    -v PROMBENCH_REPO:$PROMBENCH_REPO \
-    -f manifests/prow/components
+cat $AUTH_FILE | base64 -w 0
```

Comment:
This is needed, otherwise we'll  have to base64 encode it in the container. passing the base64 encoded file data directly helps us skip that additional step.

Reply:
Yep, noted, I updated my comment change suggestion